utils/README.md

```diff
@@ -0,0 +1,78 @@
+# **Modules Documentation**
+
+
+## **[Metrics](metrics.py)**
+Metrics are used to measure the quality of the statistical or the deep learning model. Evaluating deep learning models or algorithms is essential for any project. There some metrics available in **GDL**.
+
+### Pixel based Metrics
+
+#### Intersection over Union (IoU)
+
+The IOU metric is a pixel based metric which measures overlap using the number of pixels common between the ground truth and the predictions divided by the total pixels across both.
+<p align="center">
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\textrm{IoU}=\frac{\textrm{ground truth} \cap \textrm{prediction}}{\textrm{ground truth} \cup \textrm{prediction}}" title="\Large IoU" class="center" />
+</p>
+
+#### Dice Similarity Coefficient  (dice)
+
+The dice metric scores model performance by measuring overlap between the ground truth and predictions divided by sum of pixels of both ground truth and predictions.
```

Comment:
I see `ground truth` and `ground-truth`.  1) Please be consistent and use only one.  2) I suggest "ground truth"

Reply:
good thanks